floyd/cli/data_upload_utils.py

```diff
@@ -124,10 +124,11 @@ def complete_upload(data_config):
 
 
 def abort_previous_upload(data_config):
-    try:
```

Comment:
The purpose of this `try` statement was to catch the errors that could come up if the tarball path doesn't exist. We should either update the exceptions that we rescue or just remove the try statement